arangodb-net-standard/AqlFunctionApi/Models/GetQueryRulesResponseFlags.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace ArangoDBNetStandard.AqlFunctionApi.Models
+{
+    public class GetQueryRulesResponseFlags
```

Comment:
Maybe add comments for these? They can be copied from the online doc:

https://www.arangodb.com/docs/3.10/http/aql-query.html#return-all-aql-optimizer-rules

```
name (string): The name of the optimizer rule as seen in query explain outputs.
flags (object): An object with the properties of the rule.
    hidden (boolean): Whether the rule is displayed to users. Internal rules are hidden.
    clusterOnly (boolean): Whether the rule is applicable in the cluster deployment mode only.
    canBeDisabled (boolean): Whether users are allowed to disable this rule. A few rules are mandatory.
    canCreateAdditionalPlans (boolean): Whether this rule may create additional query execution plans.
    disabledByDefault (boolean): Whether the optimizer considers this rule by default.
    enterpriseOnly (boolean): Whether the rule is available in the Enterprise Edition only.
```

Reply:
Done.